submissions/sprint_b/app.py

```diff
@@ -0,0 +1,18 @@
+from sprint_b import app
```

Comment:
Missing docstrings for module/classes/functions
- [Description](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings)
- [Example](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)